core/frontend/src/components/scanner/servicesScanner.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <span />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import axios from 'axios'
+import { getModule } from 'vuex-module-decorators'
+import ServicesScannerStore from '@/store/servicesScanner'
+import {Service} from '@/types/SERVICE'
+
+const servicesHelper: ServicesScannerStore = getModule(ServicesScannerStore)
+
+/**
+ * Actual scanner for running services.
+ * This periodically fetches /helper/latest/web_services
+ * and updates the ServiceHelperStore with the available services
+ * @displayName Services Scanner
+ */
+export default Vue.extend({
+  name: 'ServicesFetcher',
+
+  mounted () {
+    // Fetch network data
+    this.requestData()
+
+    // Fetch periodic API request
+    this.startPeriodicRequest()
+  },
+  methods: {
+    startPeriodicRequest () {
+      setInterval(() => {
+        this.requestData()
+      }, 5000)
+    },
+    requestData () {
+      axios.get('http://localhost/helper/latest/web_services').then((response) => {
```

Comment:
localhost ?
Shouldn't this be current host ?

Reply:
I used that for running locally on port 8080 but talking to the helper in a docker, fixed.